### DISCUSSION TITLE
Cannot connect to a node added to my teleport cluster

### DISCUSSION THREAD

**Discussion by rahul-singh-yadav** [gravitational/teleport#7311] [Q&A]
Hello,

I've setup my teleport auth server with the following `teleport.yaml`:

`teleport:
  nodename: ip-172-31-0-170
  data_dir: /var/lib/teleport
  log:
    output: stderr
    severity: INFO
  ca_pin: ""
auth_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3025
  cluster_name: teleport.scoolfeed.in
ssh_service:
  enabled: "yes"
  labels:
    env: bastion-server
  commands:
  - name: bastion-host
    command: [hostname]
    period: 1m0s
proxy_service:
  enabled: "yes"
  listen_addr: 0.0.0.0:3023
  web_listen_addr: :443
  public_addr: teleport.scoolfeed.in:443
  https_keypairs: []
  acme:
    enabled: "yes"
    email: <redacted for privacy>`

---

**My current setup:**

- Teleport bastion host running inside a VPC with NSGs on aws
-  Using cloudflare as a dns proxy with an A record pointing to my teleport auth server.

**Issue**

- [x] Node token created
- [x]  Used teleport start command with flags "roles=node", "token", "ca-pin", "auth-server=uri:443"
- [x] Nodes shows up in nodes list and on web UI

When try to connect via user "ubuntu"@(my node ip) throws error "disconnected, connection timed out".

Any help is much appreciated.

Thank You for your time.

**PFA**
<img width="671" alt="Screenshot 2021-06-16 010754" src="https://user-images.githubusercontent.com/58131418/122113113-4e7cdf00-ce3f-11eb-9ef8-066a12b3c205.png">

**Comment by webvictim** [comment]
What's the output of `tctl nodes ls` on your Teleport auth server?
